scss/_utilities.scss

```diff
@@ -39,6 +39,66 @@ $utilities: map-merge(
       property: position,
       values: static relative absolute fixed sticky
     ),
+    "top": (
+      property: top,
+      values: (
+        0: 0,
+        50: 50%,
+        100: 100%
+      )
+    ),
+    "bottom": (
+      property: bottom,
+      values: (
+        0: 0,
+        50: 50%,
+        100: 100%
+      )
+    ),
+    "left": (
+      property: left,
+      values: (
+        0: 0,
+        50: 50%,
+        100: 100%
+      )
+    ),
+    "right": (
+      property: right,
+      values: (
+        0: 0,
+        50: 50%,
+        100: 100%
+      )
+    ),
+    "transform-translate-x-y": (
+      property: transform,
+      class: tt-xy,
```

Comment:
Brainstorming here...

- `translate-x-y`?
- `translate-middle`?
- `translate-center`?

Reply:
Yup, we need to work on the naming 😅

Reply:
I've updated the utility name to `translate-middle` as discussed.